<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MdrFwUForm</class>
 <widget class="QMainWindow" name="MdrFwUForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Выбор и задание скорости коммуникационного порта</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4">
       <item>
        <widget class="QComboBox" name="portBaudrateCB">
         <property name="currentIndex">
          <number>7</number>
         </property>
         <item>
          <property name="text">
           <string>1200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>230400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>460800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>921600</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="portNameCB"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="openFilePB">
      <property name="text">
       <string>Файл прошивки</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="fwfnLE">
      <property name="placeholderText">
       <string>Выберите файл прошивки</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="flasherfnLE">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Оставьте поле пустым для использования стандартного  флешера</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLabel" name="sL">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Ход работы</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Микроконтроллер</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QGroupBox" name="taskGB">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Задачи</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="dumpCB">
         <property name="text">
          <string>Считать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="eraseCB">
         <property name="text">
          <string>Стереть</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="programCB">
         <property name="text">
          <string>Программировать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="verifyCB">
         <property name="text">
          <string>Сравнить</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="runCB">
         <property name="text">
          <string>Запустить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="dumpfnLE">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Введите имя файла для сохранения прошивки</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" rowspan="2">
     <widget class="QPushButton" name="dumpfnPB">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Файл дампа</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QPushButton" name="startStopPB">
      <property name="text">
       <string>Start</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="mcuSelectCB"/>
    </item>
    <item row="13" column="0" colspan="2">
     <widget class="QPlainTextEdit" name="t">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QPushButton" name="flasherfnPB">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Файл флешера</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>35</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_Qt"/>
    <addaction name="about"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Qt">
   <property name="text">
    <string>О Qt</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>openFilePB</tabstop>
  <tabstop>fwfnLE</tabstop>
  <tabstop>portBaudrateCB</tabstop>
  <tabstop>portNameCB</tabstop>
  <tabstop>dumpCB</tabstop>
  <tabstop>eraseCB</tabstop>
  <tabstop>programCB</tabstop>
  <tabstop>verifyCB</tabstop>
  <tabstop>runCB</tabstop>
  <tabstop>dumpfnPB</tabstop>
  <tabstop>dumpfnLE</tabstop>
  <tabstop>flasherfnPB</tabstop>
  <tabstop>flasherfnLE</tabstop>
  <tabstop>startStopPB</tabstop>
  <tabstop>t</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
